c++/include/orc/Writer.hh

```diff
@@ -290,6 +290,17 @@ namespace orc {
      * @return if not set, return default value which is 64 KB.
      */
     uint64_t getMemoryBlockSize() const;
+
+    /**
+     * Set whether the compression block should be aligned to row group boundary.
```

Comment:
Add a comment to provide the expectation that boolean type might not be well aligned due to the `Boolean RLE encoding` requires packing bits into bytes. Otherwise the spec will be broken.